packages/core/src/components/radio-checkbox/_radio-checkbox.scss

```diff
@@ -1,6 +1,12 @@
+@import '../../global/mixins/exports';
+@import '../../global/variables';
+
 @include exports('ray-radio-checkbox') {
-  $ray-radio-checkbox-border-width: rem(3px);
+  $ray-radio-checkbox-border-width: rem(1px);
```

Comment:
please use the variable for this [`$ray-border-width`](https://github.com/wework/ray/blob/a235fbab2614a0f4309f4d8b0c932a7c60999536/packages/core/src/global/_variables.scss#L19)

Reply:
fixed!